### ISSUE TITLE
Weird Mesh from provided Colab example

### ISSUE THREAD

**Issue by antoinekeller** [magicleap/Atlas#4]
After running the colab provided example, it came out a weird result (see attached screenshot from cloudcompare). I didnt get any error apart from 

`# run inference
!python inference.py --model results/release/semseg/final.ckpt --scenes data/sample/sample1/info.json --voxel_dim 208 208 80

0 1 sample sample1 0 250
/usr/local/lib/python3.6/dist-packages/detectron2/modeling/backbone/fpn.py:133: UserWarning: Mixed memory format inputs detected while calling the operator. The operator will output channels_last tensor even if some of the inputs are not in channels_last format. (Triggered internally at  /pytorch/aten/src/ATen/native/TensorIterator.cpp:924.)
  prev_features = lateral_features + top_down_features
0 1 sample sample1 25 250
0 1 sample sample1 50 250
0 1 sample sample1 75 250
0 1 sample sample1 100 250
0 1 sample sample1 125 250
0 1 sample sample1 150 250
0 1 sample sample1 175 250
0 1 sample sample1 200 250
0 1 sample sample1 225 250`

Is this warning the root of the problem? How can I solve it?

Can anyone show the expected result from this example? Thx a lot

![Screenshot from 2020-08-17 11-00-08](https://user-images.githubusercontent.com/20809384/90378126-41555f00-e079-11ea-8549-011134ffddd3.png)

**Comment by zmurez-ml** [comment]
That looks like the expected result. The problem is there is no shading in your visualization which makes it hard to see the geometry (also make sure to view using backface culling since Atlas often produces closed meshes). Screenshoot below with flat shading and the result from COLMAP for comparison.

Atlas:
<img width="1142" alt="Screen Shot 2020-08-18 at 12 35 29 PM" src="https://user-images.githubusercontent.com/68235986/90557764-de82c700-e14f-11ea-968e-72f987857bab.png">

COLMAP:
<img width="1140" alt="Screen Shot 2020-08-18 at 12 36 08 PM" src="https://user-images.githubusercontent.com/68235986/90557757-daef4000-e14f-11ea-8141-3fd23c2d90df.png">

**Comment by antoinekeller** [comment]
Thank you so much for your answer. Effectively, I had to able the shading option. But, I wonder now, why are my RGB colors so strange? In the video, the floor is kind of orange/red (carpet), but it appears green in my visualization??

![Screenshot from 2020-08-19 10-00-32](https://user-images.githubusercontent.com/20809384/90608711-4c3bfb00-e203-11ea-8cc4-a4185841b4c0.png)
![Screenshot from 2020-08-19 10-00-11](https://user-images.githubusercontent.com/20809384/90608717-4cd49180-e203-11ea-80a2-b9f37fbab04d.png)

**Comment by zmurez-ml** [comment]
The colors represent the semantic labels (using the NYU40 colormap). So green is floor, pink is table, yellow is chair, etc.

The code also supports a model the regresses rgb color but we do not provide a pretrained model with this head.https://github.com/magicleap/Atlas/blob/7fe755cd2ad31348a796cf215778f58e0c835821/atlas/heads3d.py#L211

**Comment by chetana02** [comment]
Hey, can you please share the .ply file that you got after running inference.py.  Despite doing the same execution, I'm not able to open that file, as there are no vertices generated in the .ply file.

**Comment by HaFred** [comment]
Same here. And I am also wondering why is it done with `--voxel_dim 208 208 80` in the readme for inference. Whereas, in the `~/atlas/config.py`, the voxel configs are as below:
```
_C.VOXEL_DIM_TRAIN = [160,160,64]
_C.VOXEL_DIM_VAL = [256,256,96]
_C.VOXEL_DIM_TEST = [416,416,128]
```
Which one is correct for inference? Did you solve it? @antoinekeller Thank you very much.

**Comment by HaFred** [comment]
> That looks like the expected result. The problem is there is no shading in your visualization which makes it hard to see the geometry (also make sure to view using backface culling since Atlas often produces closed meshes). Screenshoot below with flat shading and the result from COLMAP for comparison.
> 
> Atlas:
> <img alt="Screen Shot 2020-08-18 at 12 35 29 PM" width="1142" src="https://user-images.githubusercontent.com/68235986/90557764-de82c700-e14f-11ea-968e-72f987857bab.png">
> 
> COLMAP:
> <img alt="Screen Shot 2020-08-18 at 12 36 08 PM" width="1140" src="https://user-images.githubusercontent.com/68235986/90557757-daef4000-e14f-11ea-8141-3fd23c2d90df.png">

BTW, how do you visualize this may I ask? Is it trimesh or cloudcompare?

**Comment by HaFred** [comment]
Hi @zmurez-ml , sorry I am a bit confused. May I know is the NYU40 dataset actually the NYUv2 in ScanNet dataset? According to the [NYUv2](https://cs.nyu.edu/~silberman/papers/indoor_seg_struct_light.pdf) paper, it seems like the table is not pink but orange?
![image](https://user-images.githubusercontent.com/13991298/123276416-bb772f80-d537-11eb-9363-3d24549e1346.png)

**Comment by dyy0205** [comment]
hi, can i ask which software did you use to show .ply file? thanks

**Comment by HaFred** [comment]
> hi, can i ask which software did you use to show .ply file? thanks

Meshlab
